riscv-asm.md

```diff
@@ -449,6 +449,36 @@ as seen by `objdump`:
   14:	00028067          	jr	t0 # 0x10
 ```
 
+Branches
+--------------------
+
+Unconditional branches are implemented by the `j(al)?r?` pseudoinstructions.
+(The underlying instructions are `jalr?`.)
+The `j(al)?` targets can be any symbol or address.
```

Comment:
Can it really be any symbol/address? I haven't tried them all.

Reply:
“Any” is indeed an overstatement, since the call/tail macros have a maximum displacement of roughly 2 GB.

Reply:
Thanks. I'll read up on this and improve it.